src/app/profile/page.tsx

```diff
@@ -6,16 +6,21 @@ import { ThemeToggler } from '@/components/ui/theme-toggler'
 
 export default function Profile() {
   return (
-    <AuthContent>
-      <ImageCard className="-my-[4.375rem] hidden sm:flex py-32 px-[3.25rem]">
+    <AuthContent className="profile-page bg-white md:bg-gradient-blue-white">
+      <h3 className="text-lg font-medium text-center py-[0.4375rem] md:hidden">
+        Profile
+      </h3>
```

Comment:
Use `title-sm` class from `globals.css`. Also, use `h2` tag instead of `h3`

```suggestion
      <h2 className="title-sm text-center md:hidden">Profile</h2>
```